src/foam/nanos/dig/DIG.js

```diff
@@ -173,6 +177,26 @@ foam.CLASS({
       view: { class: 'foam.nanos.dig.ResultView' },
 //       view: { class: 'foam.u2.tag.TextArea', rows: 5, cols: 120 },
       visibility: 'RO'
+    },
+    {
+      name: 'exportConfigArray',
+      value: [],
+      hidden: true
+    },
+    {
+      name: 'exportConfigAddOns',
+      factory: function() {
+        this.__context__.extraConfigAddOnDAO.where(this.ContainsIC.create({ arg1: this.AddOn.CONFIG_FOR_CLASS, arg2: this.cls_.id})).select().then((v) => {
```

Comment:
Better to add:
implements: [ 'foam.mlang.Expressions' ],
And then just do this.CONTAINS_IC( this.AddOn.CONFIG_FOR_CLASS, this.cls_.id )

Reply:
fixed